whylog/tests/tests_log_reader/test_files/002_match_latest/rules.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 causes: [cause]
 effect: effect
-constraints:
-    - {type: latest}
+constraints: []
+# unsupported constraint type
+#    - {type: latest}
```

Comment:
I think you can left this uncommented. We will create such constraint and will think about checkin it up. We'll probably have to discuss this.